rust/parser/mod.rs

```diff
@@ -339,50 +335,37 @@ fn visit_variable_concept(ctx: Rc<Variable_conceptContext>) -> ParserResult<Conc
 }
 
 fn visit_variable_type(ctx: Rc<Variable_typeContext>) -> ParserResult<TypeVariable> {
-    let mut var_type = match visit_type_any(ctx.type_any().unwrap())? {
-        Type::Variable(p) => p,
-        Type::Label(p) => UnboundVariable::hidden().type_(p)?,
-    };
+    let mut var_type = visit_type_any(ctx.type_any().unwrap())?.into_type_variable();
```

Comment:
`Type` now has a helper function that will turn it into a `TypeVariable`, eliminating this repeated `match` construct.